### ISSUE TITLE
Make a smaller version of poster

### ISSUE THREAD

**Issue by cvvergara** [pgRouting/osgeo#26]
I want to make also a smaller pdf version of the poster. I am thinking of an A2 size poster.

http://www.papersizes.org/a-paper-sizes.htm

Opinions

**Comment by jmckenna** [comment]
Yes I was thinking this also.  A normal page size, A2.  Similar to the work we all did for the project flyers for FOSS4G-Paris and Boston recently.   I think the flyer-size could be just the above part (not include the lower section of "Be Proactive" "Proposal" etc.  

I'd also like to see the source of this shared, preferably in something like LibreOffice or Gimp, so we can all modify it locally.  (there was a long debate on this last month when we were creating the flyers....the only software I personally was able to successfully generate SVG flyers easily with was with LibreOffice).  Well whatever software is chosen for this, the template should be stored in its raw project file for that software, so others can open and make changes quickly.  thanks!

**Comment by jmckenna** [comment]
in fact it was with LibreOffice Draw, that I used to do this easily.  It was a crazy long debate, just use whatever you prefer, but share the raw project files in the repository.  thanks!

**Comment by jmckenna** [comment]
i'm not a paper size expert.   whatever size the OSGeo flyers were.   letter i believe yes.

**Comment by jmckenna** [comment]
my comments back then: http://osgeo-org.1560.x6.nabble.com/please-create-a-new-project-flyer-for-your-project-with-the-new-branding-till-FOSS4G-Boston-tp5329556p5330591.html

**Comment by jmckenna** [comment]
size was A4 it seems :)   brain forgets easy.

**Comment by jmckenna** [comment]
Regina also used LibreOffice Draw it seems (.odg template file).  I found it to be the easiest to generate an SVG.  (example of one of my flyer templates: http://www.gatewaygeomatics.com/dl/gdal_a4.odg)

**Comment by cvvergara** [comment]
Ok, so I'll try to use libre-office draw instead of inskape and see what happens

**Comment by jmckenna** [comment]
use whatever you prefer :) That was a long debate and there was no clear winner.  I guess my point is to share the template files for whatever you choose.  thanks!

**Comment by jmckenna** [comment]
in that long thread i thought it was discovered that inkscape outputs invalid SVG......             it was a long thread sorry ha.   give it a glance and choose what you prefer.

**Comment by cvvergara** [comment]
Opening the svg with draw removes the layers.

**Comment by jmckenna** [comment]
stay with what you prefer, and check that thread to make sure your final SVG is valid.  thanks!
